deploy/evs-csi-plugin/cloud-config

```diff
@@ -0,0 +1,10 @@
+[Global]

```

Comment:
move this file to `deploy /` as a public configuration file

Reply:
done